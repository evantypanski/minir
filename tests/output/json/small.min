func main() -> none {
  let i = 0
  ret
}
