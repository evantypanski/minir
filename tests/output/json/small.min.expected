{"decls":[{"function":{"name":"main","elements":[{"stmt_kind":{"id":{"name":"i","val":{"val_kind":{"int":0},"loc":{"start":32,"end":33}},"ty":null}},"label":null,"loc":{"start":24,"end":33}},{"stmt_kind":{"ret":null},"label":null,"loc":{"start":37,"end":40}}],"params":[],"ret_ty":{"none":{}}}}]}